hera_sim/config/HERA_H1C_CONFIG.yaml

```diff
@@ -23,13 +23,13 @@ noise:
     sky_noise_jy:
         inttime: 10.7
     thermal_noise:
-        Tsky_mdl:
+        Tsky_mdl: !Tsky
```

Comment:
I agree that the semantic version is better, especially for things like this. Getting that working will require substantial effort though, and I'm viewing the state of `hera_sim` after this PR as something of an outline of all the features we would like to have, even if they don't exactly work how they're supposed to (the `Simulator.add_foregrounds` method is one example of the idea being there but not working correctly) or aren't implemented the best way they can be (the `_Defaults` class as an example here). So I'm thinking it's good to be aware of this, but to save the real effort for making the `v1.0.0` release, since that seems more along the lines of "let's actually implement everything the right way and the best way we can think of".